### PR TITLE
Use jquery v3.5.1 instead of v1.11.1

### PR DESCRIPTION
**Pull request by hoodmane**
Oddly enough `https://code.jquery.com/jquery-latest.js` does not point to the latest version of jquery, it points to version 1.11.1 (released in 2014). The jsdelivr link `https://cdn.jsdelivr.net/npm/jquery` points to 3.5.1. The terminal seems to work better with the up to date version of jquery, despite the fact that terminal docs suggested using the jquery-latest link.

Closes https://github.com/iodide-project/pyodide/issues/1276

### CODE REVIEW

File: src/templates/console.html [hunk 1]
```diff
@@ -69,28 +70,8 @@
...
-          lines[0] = this._prompt_tail + lines[0];
-          this._prompt_tail = lines.pop();
-          this.set_prompt(this._prompt_tail + prompt);
-          if( lines.length )
-            this.echo(lines.join('\n'));
-        }).bind(term);
-
-        pyconsole.stdout_callback = term.echo_no_newline;
-        pyconsole.stderr_callback = (s) => term.error(s.replace(/\n+$/, ""));
+        pyconsole.stdout_callback = (s) => term.error(s, {newline : false});
```
Comment (rth) [inline src/templates/console.html:73]: Is it OK to use the error method for the stdout though?

Comment (hoodmane) [reply src/templates/console.html:73]: Oops, typo.

**Review by rth — APPROVED** [review]
Great!

**Comment by casatir** [comment]
Thanks @hoodman, I was not able to make this `echo_newline.js` work. Probably because of the JQuery version.

Unfortunately, errors are not always displayed in red:

``` python
Welcome to the Pyodide terminal emulator 🐍
Python 3.8.2 (default, Feb 24 2021 21:10:02) on WebAssembly VM
Type "help", "copyright", "credits" or "license" for more information.
>>> 1/0
Traceback (most recent call last):
  File "<console>", line 1, in <module>
ZeroDivisionError: division by zero   <<< displayed white
>>> 1+1  <<< hitting enter here makes previous error displayed in red!
2
```

Upstream bug?

**Comment by hoodmane** [comment]
> because of the JQuery version.

Yes, there was definitely a version problem.

> Upstream bug?

Yes, jterm-newline uses a hack where it puts the temporary stuff into the prompt, but jterm prompt has a bug where it doesn't seem to display `terminal-error` formatted prompts as red. We could do `term.echo(`[[;red;]${"abc"}]`, {newline : false});` instead.

**Pull request merged**